docs/README.md

```diff
@@ -1,6 +1,6 @@
 ## Table of contents
 * [Quick Start](quick_start.md)
-* [Testing frameworks support](test_frameworks_support.md)
+* [Testing framework support](test_frameworks_support.md)
```

Comment:
Since we may have more frameworks supported in future, I was using the plural form. 
Do you find it grammatically incorrect?
Or maybe we can change it to "Supported test frameworks" like [here](https://docs.microsoft.com/en-us/appcenter/test-cloud/supported-frameworks)?

Reply:
"Testing frameworks support" sounds odd to my ears.

I'd either go with "Testing framework support" or "Supported testing frameworks".

In Java-land, we typically refer to them as "testing frameworks" instead of "test frameworks".